### ISSUE TITLE
Textbox with floating label doesn't work with ChangeDetectionStrategy.OnPush

### ISSUE THREAD

**Issue by wanton7** [telerik/kendo-angular#976]
## I'm submitting a...

* Bug report 

## Current behavior

When using following with ChangeDetectionStrategy.OnPush floating label doesn't move to top of text input.
```
<kendo-textbox-container floatingLabel="First name">
    <input kendoTextBox />
</kendo-textbox-container>
```

![onpushtextboxbreak1](https://user-images.githubusercontent.com/1009930/31254550-cc10abc6-aa31-11e7-847a-998ebc9c56eb.png)
After focus and writing some text
![onpushtextboxbreak2](https://user-images.githubusercontent.com/1009930/31254553-cf7c2baa-aa31-11e7-9564-b4922408ca34.png)

## Expected behavior
I would expect floating label to move top of text input

## Minimal reproduction of the problem with instructions
Use snippet above and set component change detection to OnPush

## What is the motivation or use case for changing the behavior?
To make floating label work with OnPush

## Environment

Package versions:
```
+-- @angular/animations@4.4.4
+-- @angular/cli@1.4.4
+-- @angular/common@4.4.4
+-- @angular/compiler@4.4.4
+-- @angular/compiler-cli@4.4.4
+-- @angular/core@4.4.4
+-- @angular/forms@4.4.4
+-- @angular/http@4.4.4
+-- @angular/language-service@4.4.4
+-- @angular/platform-browser@4.4.4
+-- @angular/platform-browser-dynamic@4.4.4
+-- @angular/router@4.4.4
+-- @progress/kendo-angular-dateinputs@1.2.0
+-- @progress/kendo-angular-dropdowns@1.2.1
+-- @progress/kendo-angular-excel-export@1.0.4
+-- @progress/kendo-angular-grid@1.4.2
+-- @progress/kendo-angular-inputs@1.3.0
+-- @progress/kendo-angular-intl@1.2.2
+-- @progress/kendo-angular-l10n@1.0.4
+-- @progress/kendo-data-query@1.0.7
+-- @progress/kendo-drawing@1.4.0
+-- @progress/kendo-theme-default@2.42.2
+-- @types/jasmine@2.5.54
+-- @types/jasminewd2@2.0.3
+-- @types/node@6.0.88
+-- codelyzer@3.0.1
+-- core-js@2.5.1
+-- jasmine-core@2.6.4
+-- jasmine-spec-reporter@4.1.1
+-- karma@1.7.1
+-- karma-chrome-launcher@2.1.1
+-- karma-cli@1.0.1
+-- karma-coverage-istanbul-reporter@1.3.0
+-- karma-jasmine@1.1.0
+-- karma-jasmine-html-reporter@0.2.2
+-- namespace-css@0.1.3
+-- protractor@5.1.2
+-- rxjs@5.4.3
+-- ts-node@3.0.6
+-- tslint@5.3.2
+-- typescript@2.5.3
+-- web-animations-js@2.3.1
`-- zone.js@0.8.18
```
Browser:
- Chrome (desktop) version 61.0.3163.100 (Official Build) (64-bit)

System:
- TypeScript version: 2.5.3
- Node version: v8.4.0 npm 5.0.4
- Platform:  Windows 10 64bit

**Comment by gyoshev** [comment]
Thank you for reporting this, we have reproduced it in the described way.

**Comment by ciel** [comment]
Is there a projected fix version for this? Unfortunately, the floating label feature is basically inert while it exists.

**Comment by gyoshev** [comment]
The fix has just been released in version `1.3.2`. Upgrade the angular-inputs package to resolve the problem.

**Comment by montella1507** [comment]
@gyoshev  bug is BACK

https://stackblitz.com/edit/angular-n4kvdl?file=app/app.component.ts

(None works OnPush doesnt on click - and blur)

**Comment by siyka-marinova** [comment]
Fix for this issue is available in the [latest dev. version](https://www.telerik.com/kendo-angular-ui/components/installation/development-builds/#toc-obtaining-development-builds) of `
kendo-angular-inputs`

**Comment by siyka-marinova** [comment]
The fix is now also available in `kendo-angular-inputs v6.4.1` and `kendo-angular-label v2.1.1`
